corporate/views/upgrade.py

```diff
@@ -354,7 +354,7 @@ def sponsorship(
         send_email(
             "zerver/emails/sponsorship_request",
             to_emails=[FromAddress.SUPPORT],
-            from_name="Zulip sponsorship",
+            from_name=f"{settings.SERVICE_NAME} sponsorship",
```

Comment:
Let's keep this hardcoded. You can do a prep commit to change this to "Zulip sponsorship request" with a comment noting this from name is not user-facing.